modules/account-lib/package.json

```diff
@@ -43,21 +41,23 @@
     "@bitgo/sdk-coin-rbtc": "^1.0.0",
     "@bitgo/sdk-coin-sol": "^1.0.0",
     "@bitgo/sdk-coin-stx": "^1.0.0",
+    "@bitgo/sdk-coin-trx": "^1.0.0",
     "@bitgo/sdk-coin-xtz": "^1.0.0",
     "@bitgo/sdk-core": "^1.0.1",
     "@bitgo/statics": "^6.17.0",
     "@ethereumjs/common": "^2.4.0",
     "@hashgraph/sdk": "~2.3.0",
     "@polkadot/api": "7.15.1",
     "@stablelib/hex": "^1.0.0",
+    "@solana/spl-token": "0.1.8",
```

Comment:
are these still used in account-lib? I just removed these dependencies since SOL was the only coin referencing them...

Reply:
Probably from the rebase. I'll fix that